collector/indices_settings.go

```diff
@@ -105,6 +106,21 @@ func NewIndicesSettings(logger log.Logger, client *http.Client, url *url.URL) *I
 					return val
 				},
 			},
+			{
+				Type: prometheus.GaugeValue,
+				Desc: prometheus.NewDesc(
+					prometheus.BuildFQName(namespace, "indices_settings", "creation_date"),
```

Comment:
This needs to follow Prometheus naming practices.

```suggestion
					prometheus.BuildFQName(namespace, "indices_settings", "creation_timestamp_seconds"),
```